client/state/test/initial-state.js

```diff
@@ -44,7 +44,7 @@ jest.mock( 'config', () => {
 
 	const config = () => 'development';
 
-	config.isEnabled = jest.fn( ( key ) => key === 'persist-redux' && persistReduxEnabled );
+	config.isEnabled = jest.fn( () => persistReduxEnabled );
```

Comment:
In this test, the entire mock for `config`, and all the calls to `{en,dis}ablePersistRedux`, can be safely removed. The persistence code doesn't depend on any `config` flag any more.

Reply:
Oh, you're totally right about this!